src/almanack/reporting/cli.py

```diff
@@ -0,0 +1,58 @@
+"""
+Setup process_repo_entropy CLI through python-fire
+"""
+
+import json
+import pathlib
+
+import fire
+
+from almanack.processing.processing_repositories import process_entire_repo
+from almanack.reporting.report import repo_entropy_report
+
+
+def process_repo_entropy(repo_path: str) -> None:
```

Comment:
Consider moving this function to the `report.py` module. Also see comment below.